instrumentation/jetty-8.0/src/test/groovy/JettyTestInstrumentation.java

```diff
@@ -26,21 +26,7 @@
   @Override
   public AgentBuilder instrument(final AgentBuilder agentBuilder) {
     return agentBuilder
-        // Jetty 8.0
-        .type(named("org.eclipse.jetty.server.HttpConnection"))
-        .transform(
-            new AgentBuilder.Transformer.ForAdvice()
-                .advice(
-                    named("handleRequest"), HttpServerTestAdvice.ServerEntryAdvice.class.getName()))
-        // Jetty 8.?
-        .type(named("org.eclipse.jetty.server.AbstractHttpConnection"))
-        .transform(
-            new AgentBuilder.Transformer.ForAdvice()
-                .advice(
-                    named("headerComplete"),
-                    HttpServerTestAdvice.ServerEntryAdvice.class.getName()))
-        // Jetty 9
-        .type(named("org.eclipse.jetty.server.HttpChannel"))
+        .type(named("org.eclipse.jetty.server.Server"))
         .transform(
```

Comment:
Was this causing failures, or just simplifying things?

Reply:
when i removed the conditional on creating `TEST_SPAN` spans, the existing jetty instrumentation ended up creating 2 `TEST_SPAN` spans, because it was also called under async context handler